app/assets/config/manifest.js

```diff
@@ -10,3 +10,6 @@
 //= link intl-tel-input/build/css/intlTelInput.css
 //= link intl-tel-input/build/img/flags.png
 //= link intl-tel-input/build/img/flags@2x.png
+
+//= link_tree ../../../node_modules/identity-style-guide/dist/assets/img
```

Comment:
We document this in the [design system usage instructions](https://github.com/18F/identity-style-guide#ruby-on-rails-rails), based on some [previously-observed issues](https://github.com/18F/identity-style-guide/pull/170). I'm hopeful it will resolve errors we see occasionally when developing locally:

![Screen Shot 2021-09-15 at 1 24 48 PM](https://user-images.githubusercontent.com/1779930/134069838-1f2c8c9a-78c4-483e-b2d5-50457abfc823.png)